### ISSUE TITLE
Options logging exposing username and passwords in logs

### ISSUE THREAD

**Issue by justinpage** [topliceanu/easter#6]
Hello,

We are auditing our logs for security and noticed the following being outputted from our apps:

```
info [AMQP Client] Connection established { host: 'rabbitmq-location.com',
  port: '5672',
  vhost: '/',
  login: 'PASSWORD',
  password: 'PASSWORD',
  heartbeat: 30 }
```

This is caused by the following statement:

```
    connection.on('ready', (function(_this) {
      return function() {
        _this.log('info', '[AMQP Client] Connection established', _this.options);
        return deferred.resolve(connection);
      };
```

Can we stop logging options or perhaps prune sensitive information before logging?

I'll be happy to update this with a PR, just wanted to see what your preference would be.

**Comment by topliceanu** [comment]
Hi @KLVTZ. 

Yes, that is definitely a good idea! I think it's still useful to have the host, post, vhost and heartbeat in the log line, so pruning would be the best option. This package imports `underscore` which has on [omit](http://underscorejs.org/#omit) method, it should do the job.

Thanks for raising this issue!

**Comment by topliceanu** [comment]
Also, feel free to open a PR, I will merge it ASAP!

**Comment by justinpage** [comment]
@topliceanu thanks for the quick reply; PR is ready-to-go :)

**Comment by topliceanu** [comment]
@KLVTZ Thanks for your PR, I merged it and published v0.0.3. Lmk if you find any other issues.
